governance/polls/Add GUNI-DAI-USDC as a new Vault Type - September 6, 2021.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Add GUNIV3DAIUSDC-A as a new Vault Type - September 6, 2021
```

Comment:
Turns out we're going with GUNIV3DAIUSDC1-A instead.

Reply:
Done.